realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(classes_LIST
     io.realm.internal.Table io.realm.internal.CheckedRow
     io.realm.internal.LinkView io.realm.internal.Util io.realm.internal.UncheckedRow
     io.realm.internal.TableQuery io.realm.internal.SharedRealm io.realm.internal.TestUtil
-    io.realm.log.LogLevel io.realm.log.RealmLog io.realm.Property io.realm.RealmSchema
+    io.realm.log.LogLevel io.realm.log.RealmLog io.realm.Property io.realm.RealmNativeSchema
```

Comment:
1) Where is that agreement??  ;-P  I can, totally, fix, but don't see either any examples of types whose name begins with OS or of documentation that prescribes it.
2) Move which to "internal"?  Just `RealmNativeSchema`?  I think that is doable!